CONTRIBUTING.md

```diff
@@ -93,11 +93,13 @@ The following project structure should be followed:
 - Run the project locally, make sure the server is listening to requests
 - Navigate to `https://github.com/login/oauth/authorize?client_id=<GITHUB_CLIENT_ID>`
 - Authorize the application
-- Copy the response cookie named rds-session from the redirected request <HOST>/auth/github/callback?code=<SOME_AUTOGENERATED_CODE>
+- Copy the response cookie named `rds-session` (`rds-session-development` in development mode) from the redirected request <HOST>/auth/github/callback?code=<SOME_AUTOGENERATED_CODE>
+- Once authorized, simply go to the Application tab of your browser's developer tools and boom! There's your 🍪.
```

Comment:
Isn't this confusing? We should be including **either** of these lines. Both of them are two different ways to achieve the same thing. Please keep the second line only.

We could mention about the cookie name in the second line and keep that one only 🙂 
Also, please phrase it in proper words. Inside application tab, inside the cookies section for localhost, there is the cookie. In fact, the `application` tab is valid only for chrome. The cookies in firefox are in `storage -> cookies`

Reply:
Pushed in new changes.

Reply:
Please remove line 96 😅

Reply:
Oopsie! Pushed again.